.vscode/launch.json

```diff
@@ -2,29 +2,46 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Debug",
+      "name": "Debug current file",
       "type": "node",
       "request": "launch",
-      "runtimeExecutable": "npm",
-      "runtimeArgs": ["run-script", "start", "--", "--inspect-brk=9229"],
-      "port": 9229
+
+      // Debug current file in VSCode
+      "program": "${file}",
+
+      /*
+    Path to tsx binary
+    Assuming locally installed
+    */
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/tsx",
+
+      /*
+    Open terminal when debugging starts (Optional)
+    Useful to see console.logs
+    */
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+
+      // Files to exclude from debugger (e.g. call stack)
+      "skipFiles": [
+        // Node.js internal core modules
+        "<node_internals>/**",
+
+        // Ignore all dependencies (optional)
+        "${workspaceFolder}/node_modules/**"
+      ]
     },
     {
-      "name": "Debug Server",
+      "name": "Debug All Tests (Node)",
```

Comment:
You can run this script in the Debug window, set a breakpoint in any test file, and it'll stop there correctly

Reply:
Works for me now, thank you!